app/locales/sk.json

```diff
@@ -109,5 +110,27 @@
   },
   "process": {
     "newVersionAvailable": "Je dostupná nová verzia!"
+  },
+  "welcome": {
+    "welcomeToStretchly": "Vitajte v Stretchly",
```

Comment:
Please do not use Google translate/or similar tool for translates. Those tools does not make correct translations. I would rather have English there and have it translated by someone native later.

Reply:
Okay. So should I replace all translations with English?

Reply:
Let's keep for the end, when we have final wording.

Reply:
Okay